deliveries/iris-milestone-2.md

```diff
@@ -0,0 +1,28 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** https://github.com/w3f/Grants-Program/tree/master/applications/iris.md
```

Comment:
```suggestion
* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/iris.md
```

Reply:
The link just redirects to the edit, but CI won't parse it

Reply:
I've updated the link.